tools/contract_checks/README.md

```
@@ -0,0 +1,458 @@
+    root=REPO_ROOT / 'path' / 'to' / 'python' / 'source',
+    internal_packages=frozenset({'my_internal_package'}),
+),
+```
+
+No other change is needed — the CLI's tree iteration loop picks up the new entry automatically on the next run, and the install hook recursively finds every `requirement*.txt` under `root` on its own (no per-tree requirements config).
+
+## Reading CI output
+
+Failures take the form:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)